src/Powercord/plugins/pc-moduleManager/components/parts/BaseProduct.jsx

```diff
@@ -2,12 +2,23 @@ const { React, getModule, constants: { Routes }, i18n: { Messages } } = require(
 const { Divider, Button } = require('powercord/components');
 
 const { shell: { openExternal, openPath } } = require('electron');
-
+const Path = require('path');
```

Comment:
massive nitpick, but can we make this lowercase

Reply:
i only copied every other const ;-;

Reply:
The other consts are React components, this one is a module 🦜